Bottom: 0db07887114772eb6c4e34d242dafa36bbb125c1
Top:    b2120fbe31c8a6acf15ddc260f8238ec358b9f56
Author: Johannes Rudolph <johannes_rudolph@gmx.de>
Date:   2010-05-29 16:14:49 +0200

cleanup


---

diff --git a/objectformatter/src/main/scala/net/virtualvoid/string/EnhancedStringsPlugin.scala b/objectformatter/src/main/scala/net/virtualvoid/string/EnhancedStringsPlugin.scala
index 79035ab..38d1c9c 100644
--- a/objectformatter/src/main/scala/net/virtualvoid/string/EnhancedStringsPlugin.scala
+++ b/objectformatter/src/main/scala/net/virtualvoid/string/EnhancedStringsPlugin.scala
@@ -30,6 +30,7 @@ class EnhancedStringsPlugin(val global: Global) extends Plugin {
     
     class ESTransformer(unit: CompilationUnit) extends TypingTransformer(unit) {
         localTyper = analyzer.newTyper(analyzer.rootContext(unit, EmptyTree, false))
+    
 		val it = ValDef(Modifiers(Flags.PARAM), "it", TypeTree(), EmptyTree)
       
 	    /** When using <code>preTransform</code>, each node is
@@ -58,7 +59,7 @@ class EnhancedStringsPlugin(val global: Global) extends Plugin {
 	      case AST.ToStringConversion(exp) => Select(compileExpression(exp),"toString")
 	      case AST.Expand(exp,sep,inner) => Apply(
 	    		  							   Select(
-	    		  							     Apply(Select(compileExpression(exp),"map")
+	    		  							     Apply(Select(compileExpression(exp), "map")
 	    		  							    		 ,List(Function(List(it),compile(inner)))),"mkString")
                                                ,List(Literal(Constant(sep))))
 	      case AST.Conditional(cond,thenEls,elseEls) =>
